docs/Overview.md

```diff
@@ -75,4 +75,10 @@ Because Draft.js supports unicode, you must have the following meta tag in the `
 <meta charset="utf-8" />
 ```
 
+`Draft.css` should be included when rendering the editor. Learn more about [why](/docs/advanced-topics-issues-and-pitfalls.html#missing-draft-css).
+
+```js
```

Comment:
There are many ways to include CSS, and some folks will use `require` vs. `import` or might even compile their CSS and include the bundle in a `link` tag in their HTML header. What do you think of leaving out the sample of how to import it, since that part can vary so much?